.github/workflows/ci.yaml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc-ver: ["9.2.8", "9.4.5", "9.6.2"]
-        cabal: ["3.10.1.0"]
+        ghc-ver: ["9.4.5", "9.6.2", "9.8.2"]
```

Comment:
It might be worth upgrading to the latest minor versions of each of these GHC releases (in particular, 9.4.8, 9.6.6, and 9.8.4) while we're at it.

Reply:
I agree, though I might leave that for someone with `ghcup`, I sadly am using Nix to manage my GHC due to https://github.com/NixOS/nixpkgs/issues/212716